packages/common/test/util.test.js

```diff
@@ -1,5 +1,209 @@
 import { expect } from 'chai';
-import { expandReferences, normalizeOauthConfig } from '../src/util';
+import {
+  enableMockClient,
+  expandReferences,
+  normalizeOauthConfig,
+  get,
+  post,
+  del,
+} from '../src/util';
+
+const client = enableMockClient('https://www.example.com');
+
+describe('methods', () => {
+  it('should send a GET', async () => {
+    // set up mock GET at /
+    let request;
+    client
+      .intercept({
+        path: '/api',
+        method: 'GET',
+      })
+      .reply(200, r => {
+        request = r;
+        return {};
+      });
+
+    await get('https://www.example.com/api');
```

Comment:
I think we should allow the `baseUrl` stuff to be specified in the request options (rather than have to require the whole url to be passed every time)

Because then at the top of our test file we can do:
```
const options = { baseUrl: 'https://www.example.com' };
const client = enableMockClient(options.baseUrl);
```
And then in each test we can do:
```
await get('/api', options)
```

Reply:
Added support for `baseUrl` option